spynnaker/pyNN/models/recording_common.py

```diff
@@ -16,13 +16,19 @@
 
 
 class RecordingCommon(object):
-    def __init__(self, population):
+    def __init__(self, population, sampling_interval=None):
         """ object to hold recording behaviour
 
         :param population: the population to record for
+        :param simulator: the spinnaker control class
         """
 
         self._population = population
+
+        self._sampling_interval = None
+        if sampling_interval is not None:
+            self._sampling_interval = sampling_interval
```

Comment:
or just ```self._sampling_interval = sampling_interval```

Reply:
Actually Worse. Not sure sampling_interval belongs here at all.

Reply:
Will look at this in the selective recording PR

Reply:
In Selective record sampling intervals are held in NeuronRecorder and no longer in RecordingCommon